adding link to list of events (#7965)
magento/devdocs: src/recommendations/verify.md

## Verify events are firing correctly

> To verify that the events used for metrics are firing correctly, look for the `impression-render`, `view`, and `rec-click` events in the Snowplow Analytics Debugger. The full list of events can be found on [this page](https://devdocs.magento.com/recommendations/events.html)

Reviewer: ```suggestion
To verify that the events used for metrics are firing correctly, look for the `impression-render`, `view`, and `rec-click` events in the Snowplow Analytics Debugger. See the [full list of events](https://devdocs.magento.com/recommendations/events.html).
```
